src/index.ts

```diff
@@ -38,10 +41,84 @@ async function parseHtmlToDocument(html: IHtml): Promise<CheerioAPI> {
   return doc;
 }
 
-fetchProductHtml(858911)
+const productList: IProduct[] = [];
+let productId = 1;
+
+const getProductList = (doc: CheerioAPI) => {
+  const title = doc(
+    "#page_product_detail > div.right_area.page_detail_product > div.right_contents.section_product_summary > span > em"
+  ).text();
+  const engTitle = doc(
+    "#page_product_detail > div.right_area.page_detail_product > div.right_contents.section_product_summary > span > span"
+  ).text();
+
+  const category = doc(
+    "#page_product_detail > div.right_area.page_detail_product > div.right_contents.section_product_summary > div.product_info > p > a:nth-child(2)"
+  ).text();
+  const categoryId = doc(
+    "#page_product_detail > div.right_area.page_detail_product > div.right_contents.section_product_summary > div.product_info > p > a:nth-child(2)"
+  )
+    .attr("href")
+    ?.split("/category/")[1];
+
+  const originalPrice = doc("#goods_price")
+    .first()
+    .text()
+    .split("원")[0]
+    .replace(",", "");
+  const price = doc("#sPrice > ul > li.pertinent > span.txt_price_member")
+    .text()
+    .split("원")[0]
+    .replace(",", "");
+
+  const brandName = doc(
+    "#page_product_detail > div.right_area.page_detail_product > div.right_contents.section_product_summary > div.product_info > p > a:nth-child(3)"
+  )
+    .text()
+    .trim()
+    .replace(/[()]/gi, "");
```

Comment:
```suggestion
    .replace(/[()]/gi, ""); // 코드에 대한 간단한 설명
```

간단하게만 설명해줘도 가독성 좋아질듯!